d2ir/import_test.go

```diff
@@ -75,6 +122,18 @@ x.d2:1:6: connection missing destination
 x.d2:1:7: connection missing source`)
 				},
 			},
+			{
+				name: "cyclic",
+				run: func(t testing.TB) {
+					_, err := compileFS(t, "index.d2", map[string]string{
+						"index.d2": "...@x",
+						"x.d2":     "...@y",
+						"y.d2":     "...@q",
+						"q.d2":     "...@x",
```

Comment:
what if this is `"...@index.d2"`? will the err msg be right?

```
x -> y -> q -> index -> x
```

Reply:
That's not the correct error message there. It would be:

```
index -> x -> y -> q -> index
```

The first import chain is what's reported.